server/src/routines/matchmaking.ts

```diff
@@ -280,6 +281,7 @@ function* gameManager(con: GameController) {
 				turnActionsBuffer,
 				con.gameCode,
 			)
+			yield* sendAfterGameInfo(gamePlayers)
```

Comment:
This should be called anyway, to make sure Evil X works with achievements for example.

Reply:
Actually, I think I got mixed up here, never mind